test/wicked_test.rb

```diff
@@ -77,4 +77,29 @@ class DummyNetwork
       expect(subject.ibft_interfaces).to eql(["eth0.42", "eth0", "eth1"])
     end
   end
+
+  describe "#firmware_interfaces" do
+    let(:stdout) { instance_double("Yast::Execute") }
+    let(:output) { "ibft    eth1 eth1.10\nibft    eth2 ibft\nnbft    eth3\nredfish usb0 usb0.42" }
```

Comment:
Although I believe the regexp is fully correct, I would like to see an example with "ibft" or "nbft" as part of the name of the device. Just as a reminder if someone changes the regexp.

```
ibft    eth1 eth1.10
nbft    eth3
nbft    nbft0 nbft0.40
```